src/domain/enrolment/EditEnrolmentPage.tsx

```diff
@@ -112,7 +114,7 @@ const EditorEnrolmentPage: React.FC = () => {
           groupSize: enrolment?.studyGroup.groupSize.toString() || '',
           groupName: enrolment?.studyGroup.groupName || '',
           name: enrolment?.studyGroup.name || '',
-          studyLevels: studyLevels || [],
+          studyLevels: map(studyLevels, method('toUpperCase')) || [],
```

Comment:
studyLevels.map(s => s.toUpperCase())?

I think we could get rid of lodash functions? This line required to looksvat lodash docs to understand.

Reply:
yea, fixed